modules/monitoring-configuring-audit-logs-for-metrics-server.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="configuring-audit-logs-for-metrics-server_{context}"]
+= Configuring audit logs for Metrics Server
+
+You can configure audit logs for Metrics Server to help you troubleshoot issues with the server. 
+Audit logs record the sequence of actions in a cluster. It can record user, application, or control plane activities.
+
+You can set audit log rules, which determine what events are recorded and what data they should include. This can be achieved with the following audit profiles:
+
+* *Metadata (default)*: This profile enables the logging of event metadata (user, timestamps, resource, verb, and others). It does not record request and response bodies.
```

Comment:
IBM Style suggests to [avoid using parentheses](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses) in general text. 

Suggestions:

```suggestion
* *Metadata (default)*: This profile enables the logging of event metadata such as user, timestamps, resource, and verb. It does not record request and response bodies.
```

```suggestion
* *Metadata (default)*: This profile enables the logging of event metadata including user, timestamps, resource, and verb. It does not record request and response bodies.
```